tests/test_submission.py

```diff
@@ -225,3 +226,40 @@ def test_sticky_unsticky(self):
         submission_b.unsticky()
         self.assertFalse(submission_a.refresh().stickied)
         self.assertFalse(submission_b.refresh().stickied)
+
+
+class OAuthSubmissionTest(PRAWTest):
```

Comment:
Should we make an OAuthPRAWTest parent class that calls `set_oauth_app_info?` in its default setUp function?